src/services/githubauth.service.js

```diff
@@ -1,3 +1,5 @@
+import fetch from 'node-fetch';
```

Comment:
is there functionality to do this through the github library. If possible, I'd like to do it there, since it means we don't have to maintain it. 

Here is the documentation to the library: https://octokit.github.io/rest.js/

Reply:
You mean do the whole exchange through this library rather than through a fetch? If so, then yes this is a viable option. Should the way it is currently implemented fail, then that would be a good time to switch how we do it.

Reply:
OK. It is fine to leave it.